deps/wazuh_testing/wazuh_testing/fim.py

```diff
@@ -2398,3 +2398,18 @@ def get_fim_mode_param(mode, key='FIM_MODE'):
         return {key: {'whodata': 'yes'}}, metadata
     else:
         return None, None
+
+
+def check_fim_start(file_monitor):
+    """Check if realtime starts, whodata starts or ends the initial FIM scan.
+
+    Args:
+        file_monitor (FileMonitor): file log monitor to detect events.
+    """
+    mode = global_parameters.current_configuration['metadata']['fim_mode']
+    if mode == 'realtime':
+        detect_realtime_start(file_monitor)
+    elif mode == 'whodata':
+        detect_whodata_start(file_monitor)
+    else:
+        detect_initial_scan(file_monitor)
```

Comment:
You should change the similar fixtures in order to use your function.

Reply:
```suggestion
def check_fim_start(file_monitor):
    """Check if realtime starts, whodata starts or ends the initial FIM scan.

    Args:
        file_monitor (FileMonitor): file log monitor to detect events.
    """
    mode = global_parameters.current_configuration['metadata']['fim_mode']
    if mode == 'realtime':
        detect_realtime_start(file_monitor)
    elif mode == 'whodata':
        detect_whodata_start(file_monitor)
    else:
        detect_initial_scan(file_monitor)
        
```

Reply:
Fixed in 1904fcb